coding-agent/src/coding_agent/ui/session_manager.py

```
@@ -413,6 +420,30 @@ def __post_init__(self) -> None:
+    def broadcast_event_nowait(self, event: dict[str, Any]) -> EventBroadcastResult:
+        active_queues: list[asyncio.Queue[dict[str, Any]]] = []
+        delivered_count = 0
+        full_pruned_count = 0
+        failed_pruned_count = 0
+
+        for queue in self.event_queues:
+            try:
+                queue.put_nowait(event)
+            except asyncio.QueueFull:
+                full_pruned_count += 1
+            except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)